djangocms_picture/models.py

```diff
@@ -23,19 +23,38 @@
 
 # add setting for picture alignment, renders a class or inline styles
 # depending on your template setup
-PICTURE_ALIGNMENT = getattr(
-    settings,
-    'DJANGOCMS_PICTURE_ALIGN',
-    (
-        ('left', _('Align left')),
-        ('right', _('Align right')),
-        ('center', _('Align center')),
+def get_alignment():
+    alignment = getattr(
+        settings,
+        'DJANGOCMS_PICTURE_ALIGN',
+        (
+            ('left', _('Align left')),
+            ('right', _('Align right')),
+            ('center', _('Align center')),
+        )
     )
-)
+    return alignment
```

Comment:
Hmm, what's the advantage of using a function for this rather than the module-level variable? As far as I can see, outside of tests, this is only ever called at module level, and even in tests, the runtime calls do not involve any settings overrides.

Reply:
It's called inside the models itself and the migrations, similar to get_templates. The main reason to change it was to be able to test it's values.